hw/ip/prim/fpv/prim_arbiter_fpv.core

```diff
@@ -0,0 +1,25 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:fpv:prim_arbiter_fpv:0.1"
```

Comment:
Hi Michael, I saw the rest of the fpv core file does not have the `:0.1` postfix. I just wonder if this one is necessary, if so, we will have to change the `fpv_all` to support this :) And in general, I wonder what is the naming convention for the fusesoc core file?